docs/rules/dynamic-import-chunkname.md

```diff
@@ -0,0 +1,66 @@
+# dynamic imports require a leading comment with a webpackChunkName (dynamic-import-chunkname)
+
+This rule reports any dynamic imports without a webpackChunkName specified in a leading block comment in the proper format.
+
+This is a useful rule because if the webpackChunkName is not defined in a dynamic import, Webpack will autogenerate the chunk name.
+
+## Rule Details
+This rule runs against `import` by default, but can be configured to also run against an alternative dynamic-import function, e.g. 'dynamicImport.'
```

Comment:
`import` here should probably be `import()` to differentiate it as a dynamic import.